content/blog/2022-07-26-iterative-studio-model-registry.md

```diff
@@ -0,0 +1,185 @@
+production-readiness. A version may be running in production, another version
+that seems to perform better may be in staging, and a couple more versions could
+be in active development by you and your teammates - using updated
+hyperparameters, datasets, or algorithms.
+
+How do you keep track of all your models, their versions, and deployment
+statuses? How do you get answers to questions like these easily:
+
+- Which model version is currently in production?
+- When was the last time this model was updated?
+
+If you are like some of the data scientists we know, you may have a Google sheet
+or a Notion page with the list of all your models, their changes, deployment
+history, and so on. But this is highly error-prone and will probably get
+out-of-date very quickly. Or maybe you upload all your models to a cloud bucket
+and “attach” text reports to them. Not very maintainable or searchable either.
+We’ve even seen people use sticky notes, or better yet, rely on their memory 😀
+
+Some of the more organized folks use Model Registries - tools created
+specifically to organize models into a central, searchable repository. While
+this is definitely better than using random files or sticky notes, one major
+problem persists: the data science and machine learning team members work
+completely isolated from the software development and DevOps team members. This
+makes collaboration far more time consuming than it should be.
+
+![Teams can work in disconnected silos](/uploads/images/2022-07-26/disconnected-silos.png)
+
+Some even implement in-house systems, and maybe you are also planning to do so.
+But these can get expensive to develop and maintain.
+
+This is a common problem, and deserves a robust, intuitive and scalable
+solution.
+
+**We built the Iterative Studio Model Registry to solve these problems.**
+
+Iterative Studio Model Registry enables ML teams to collaborate on models by
+providing model organization, discovery, versioning, lineage (tracing the origin
+of the model), and the ability to manage deployment statuses such as,
+development, staging, and production across multiple projects.
+
+## Utilize your existing Git infrastructure
+
+Iterative Studio Model Registry is built on top of Git, which means:
+
+- You can reuse your existing Git infrastructure to manage ML models together
+  with code, data, experiment pipelines, and deployment statuses.
+- You can use GitOps for model deployment, and trigger model deployment from
+  Iterative Studio, which you can also use to run your ML experiments.
+- Your DS/ML folks and Software/DevOps folks can work together more easily,
+  because they utilize the same tools and infrastructure.
+
+## Open MLOps
+
+A core philosophy at Iterative is open MLOps - we build tools that work with
+your infrastructure. Our toolstack is modular, so you can build your model
+registry on top of your existing cloud and DevOps infrastructure and avoid
+vendor lock-in.
```

Comment:
This is a little awkward because we just plopped it in here and it's not a part of the problem statement at all.  I agree it's a good point it should be here.  If we think that it's just understood what vendor lock-in is, then I guess it's ok.  But it feels a little like "where did that come from?" when you read through.

Reply:
Removed ` and avoid vendor lock-in`. I think that was causing the issue. But if it's still not fitting well, let me know and I'll work on it again.